recipes/libjpeg-turbo/all/CMakeLists.txt

```diff
@@ -5,4 +5,9 @@ project(cmake_wrapper)
 include(conanbuildinfo.cmake)
 conan_basic_setup(NO_OUTPUT_DIRS)
 
+if(NOT CMAKE_SYSTEM_PROCESSOR)
+    set(CMAKE_SYSTEM_PROCESSOR ${CONAN_LIBJPEG_SYSTEM_PROCESSOR})
+endif()
```

Comment:
After some experimentations it should be:

```
if(NOT DEFINED CMAKE_SYSTEM_PROCESSOR OR CMAKE_SYSTEM_PROCESSOR STREQUAL "")
    set(CMAKE_SYSTEM_PROCESSOR ${CONAN_LIBJPEG_SYSTEM_PROCESSOR})
endif()

Reply:
Oops, no it's fine, it doesn't check if CMAKE_SYSTEM_PROCESSOR is defined, so it works.

Reply:
"STREQUAL" check is surely more strict, but I can't imagine any sane value for this variable which could be rejected by simple "NOT".

However there are cases in CMake when its necessary to check a variable for emptiness. It's done simply by taking it's value in the quotes:
```cmake
if("${VAR}" STREQUAL "")
```
There's no need to check if it's defined.

Reply:
Yes, this check is perfectly fine, my bad, I thought the check was `if(NOT DEFINED CMAKE_SYSTEM_PROCESSOR)`.

Reply:
Really, I tried `NOT DEFINED` first, but it didn't work, so I switched to just `NOT`.